src/components/AddBatchModule/AddBatch.jsx

```diff
@@ -237,9 +238,14 @@ const AddBatch = (props) => {
               {!batchShow ? "Switch Batch" : editId ? "Edit Batch" : "Add Batch"}
             </div>
           }
-          onClose={()=>{props.onClose() }}
+          onClose={()=>{props.onClose(),
+            setBatchshow(false)
+          }}
           open={props.open}
           placement="left"
+          maskClosable={()=>{
+            setBatchshow(false)
+          }}
```

Comment:
As per doc in this attr their expecting only boolen but you are providing like a arrow fn is that working?